packages/dx-scheduler-core/src/plugins/view-state/reducers.js

```diff
@@ -0,0 +1,5 @@
+import moment from 'moment';
+
+export const setCurrentDate = (currentDate, { nextDate, step, back = false }) => (
```

Comment:
According to our guidelines when a method should follow only one semantic. 'Set' when setting a direct input value or 'change' when modify current value. Please, use select what you want the method do and call them correctly. If you need 'set' and 'change' semantics at one method, please separate it to two methods